packages/next-on-pages/docs/caching.md

```diff
@@ -0,0 +1,52 @@
+# Caching and Data Revalidation
+
+`@cloudflare/next-on-pages` comes with support for data revalidation and caching for fetch requests. This is done in our router and acts as an extension to Next.js' built-in functionality.
+
+## Storage Options
+
+There are various different bindings and storage options that `@cloudflare/next-on-pages` supports for caching.
+
+We recommend evaluating each option and choosing the one that best suits your use case, depending on latency and consistency requirements.
+
+### Workers KV
+
+[Workers KV](https://developers.cloudflare.com/workers/learning/how-kv-works/) is globally-distributed, low-latency storage option that is ideal for caching data. While it's designed for this use case, KV is an eventually-consistent data store, meaning that it can take up to 60 seconds for changes to propagate globally. This is fine for many use cases, but if you need to ensure that data is updated more frequently globally, you should consider a different storage option.
+
+1. Create a [new KV Namespace](https://dash.cloudflare.com/?to=/:account/workers/kv/namespaces).
+2. Find your [Pages project](https://dash.cloudflare.com/?to=/:account/workers-and-pages) in the Cloudflare dashboard.
+3. Go to your Pages project Settings > Functions > KV Namespace Bindings.
+4. Add a new binding mapping `KV_SUSPENSE_CACHE` to your created KV Namespace.
+
+### Cloudflare D1
+
```

Comment:
I think we should warn that D1 is in alpha? (so not really stable and/or production ready)